pkg/square/builder.go

```diff
@@ -410,10 +409,15 @@ func (e element) maxShareOffset() int {
 	return e.numShares + e.maxPadding
 }
 
-func worstCaseShareIndexes(blobs, maxSquareCapacity int) []uint32 {
+// worstCaseShareIndexes returns the largest possible share indexes for a set
+// of blobs at a given appversion. Largest possible is "worst" in that protobuf
+// uses varints to encode integers, so larger integers can require more bytes to
+// encode.
+func worstCaseShareIndexes(blobs int, appVersion uint64) []uint32 {
```

Comment:
[suggestion] The function name isn't very clear about what it represents as the "worst". It might be better to use something like `shareIndexUpperBound` to make it more self-explanatory.